### DISCUSSION TITLE
RecursionError: maximum recursion depth exceeded while calling a Python object Normalization failed: type=AttributeError args=<unknown>"

### DISCUSSION THREAD

**Discussion by saidamahmoud** [castorini/pyserini#2045] [Q&A]
Thank for the repo

I begain to use your repo to be able to run hyde

i was trying to run the import 
like
`
from pyserini.search.lucene import LuceneSearcher
from pyserini.search.faiss import FaissSearcher
`
But i got Recursion error 
`
RecursionError: maximum recursion depth exceeded while calling a Python object
Normalization failed: type=AttributeError args=<unknown>"
`
any help?

**Comment by lintool** [comment]
Can you provide more details/context? For example, what version are you using? What exactly are you trying to do?
